src/content/advanced/automatic-node-termination/index.md

```diff
@@ -14,14 +14,16 @@ owner:
 
 # Automatic termination of unhealthy nodes
 
-{{< platform_support_table aws="alpha=v12.6.0" azure="alpha=v13.1.0" kvm="alpha=v14.0.0" >}}
+{{< platform_support_table aws="alpha=v12.6.0" aws="ga=v15.0.0" azure="alpha=v13.1.0" kvm="alpha=v14.0.0" >}}
 
 ## Introduction
 
 Degraded nodes in a Kubernetes cluster should be a rare issue, however when it occurs, it can have severe consequences for the workloads scheduled to the affected nodes. The goal should be to detect bad nodes early and remove them from the cluster, replacing them with healthy ones.
 
 Starting with workload cluster release v12.6.0 for AWS, v13.1.0 for Azure and v14.0.0 for KVM, you now have the option to automate the detection and termination of bad nodes. When enabled, all nodes in your cluster are periodically checked. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
 
+From AWS release `v15.0.0` the feature is **enabled by default**. On other providers it has to be enabled.
```

Comment:
```suggestion
From AWS workload cluster release v15.0.0 the feature is enabled by default. On other providers it has to be enabled explicitly.
```

We don't use literals for release versions. Also I think the emphasis is not necessary.